winmake.ps1

```diff
@@ -28,6 +28,37 @@ function Local-Machine {
     Run-Command "./test/tools/build/ginkgo.exe -vv  --tags `"$remotetags`" -timeout=90m --trace --no-color $files pkg/machine/e2e/. "
 }
 
+function Win-SSHProxy {
```

Comment:
Would you consider adding a comment for this function indicating the expected starting directory, and maybe other important assumptions?

Reason I say is b/c it's hard to guess and easy to break.